.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
This is not a necessary addition for this PR. Remove this file.

Reply:
Done